src/shared/clients/ec2Client.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EC2 } from 'aws-sdk'
```

Comment:
Should probably use v3 sdk instead.

Reply:
Will make this its own PR to make sure I don't break anything before merge.